guide/index.md

```diff
@@ -110,8 +103,10 @@ export default defineConfig({
 })
 ```
 
-If you build your application, the [Web App Manifest](https://developer.mozilla.org/en-US/docs/Web/Manifest) will be generated and configured on the application entry point, the service worker will be also generated and the script/module to register it in the browser added.
+If you build your application, the [Web App Manifest][webmanifest] will be generated and configured on the application entry point, the service worker will be also generated and the script/module to register it in the browser added.
```

Comment:
same here

Reply:
Sorry, I don't understand. The link references seem to be working fine for me with both `pnpm nr dev` and `pnpm nr preview`. Am I missing something?

Reply:
It is about the VitePress build process, I'm going to check if the links are fine in the PR preview, just forgot my comments about this.